JPEGImageCompression/image_compression.py

```diff
@@ -357,19 +364,242 @@ def update(mob, alpha):
         self.wait(3)
 
 
+class JPEGDiagram(Scene):
+    def construct(self):
+
+        # animation section
+
+        # self.intro()
+
+        # self.play(*[FadeOut(mob) for mob in self.mobjects])
+
+        self.intro_2()
+
+        self.play(*[FadeOut(mob) for mob in self.mobjects])
+
+        self.data_flow()
```

Comment:
We might need to figure out a better representation of a "compressed image" because the current figure is not clear that it represents a compressed image (at least to me).